src/server/data/automatedTestingData.js

```diff
@@ -179,14 +244,29 @@ async function generateCosineSquaredTestingData(amplitude = 1) {
 async function generateFourDayTestingData() {
```

Comment:
Update comment above about into DB. This needs to happen for a lot of functions below.